pkg/usageStats/stats.go

```diff
@@ -541,6 +520,77 @@ func GetUsageStats(pastXhours uint64, granularity UsageStatsGranularity, orgid u
 	return resultMap, nil
 }
 
+// The data format has evolved over time:
+// - Initially, it was: bytes, eventCount, time
+// - Then, metrics were added: bytes, eventCount, metricCount, time
+// - Later, logsBytesCount and metricsBytesCount were added. However, the new format is backward compatible with the old formats.
+// The current format is: bytes, eventCount, metricCount, time, logsBytesCount, metricsBytesCount
+func parseRecord(record []string) (ReadStats, error) {
+	var readStats ReadStats
+	var err error
+
+	if len(record) < 3 {
+		return readStats, fmt.Errorf("invalid record length: %d", len(record))
+	}
+
+	readStats.BytesCount, err = strconv.ParseUint(record[0], 10, 64)
+	if err != nil {
+		return readStats, err
+	}
+	readStats.EventCount, err = strconv.ParseUint(record[1], 10, 64)
+	if err != nil {
+		return readStats, err
+	}
+
+	switch len(record) {
+	case 3:
+		tsString, err := strconv.ParseInt(record[2], 10, 64)
+		if err != nil {
+			return readStats, err
```

Comment:
`fmt.Errorf("parseRecord: could not parse timestamp field: %v", tsString)` could be a better error